ftl-project.toml

```diff
@@ -14,6 +14,9 @@ disable-ide-integration = true
   [modules.echo]
     [modules.echo.configuration]
       default = "inline://ImFub255bW91cyI"
+  [modules.relay]
+    [modules.relay.secrets]
+      FTL_DSN_RELAY_EXEMPLARDB = "inline://OUNUTlo3ekxuZjZPSmpxZmNyazR3d3ZZN1NINSs4K2MwdzdPNjQ1b0dLVHB4MVJZVExNK1J3dFhlWEFDMUZEWUMzRVZzTFhIYWcyVzdQSk1wV01tcjBRVUJFMTduaGFpQjArRTNydlZpeldTMllmTmk4SWI4YmdnRkVOQksvRW0zNEtuR05nNGFJV1krUXk0VUFHZjI4ckhscWo4"
```

Comment:
Do we need the other `exemplar/ftl-project.toml` stuff here as well? Like `nonce` and `log_file`? Or should we just use this single root `ftl-project.toml` for both the `examples` and the `exemplar`?